jailer/src/env.rs

```diff
@@ -256,6 +262,25 @@ impl Env {
             return Err(Error::ChangeDevNetTunOwner(sys_util::Error::last()));
         }
 
+        #[cfg(feature = "vsock")]
+        {
+            // Do the same for /dev/vhost_vsock with (major, minor) = (10, 241).
+            if unsafe {
+                libc::mknod(
+                    dev_vhost_vsock_path.as_ptr(),
+                    libc::S_IFCHR | libc::S_IRUSR | libc::S_IWUSR,
+                    libc::makedev(10, 241),
+                )
+            } < 0
+            {
+                return Err(Error::MknodDevVhostVsock(sys_util::Error::last()));
```

Comment:
Can we use `std::io::Error` instead of `sys_util::Error`?

LE: Oh, I see plenty of `sys_util::Error` in the jailer, so this is out of scope in this PR